generators/docker-compose/index.js

```diff
@@ -108,22 +108,10 @@ module.exports = yeoman.Base.extend({
                 validate: function (input) {
                     var path = this.destinationPath(input);
                     if(shelljs.test('-d', path)) {
-                        var files = shelljs.ls('-l',this.destinationPath(input));
-                        this.appsFolders = [];
-
-                        files.forEach(function(file) {
-                            if(file.isDirectory()) {
-                                if(shelljs.test('-f', file.name + '/.yo-rc.json')) {
-                                    var fileData = this.fs.readJSON(file.name + '/.yo-rc.json');
-                                    if(fileData['generator-jhipster'].baseName !== undefined || fileData['generator-jhipster'].applicationType == 'uaa') {
```

Comment:
Just be carefull. When rebasing, you removed your test : `fileData['generator-jhipster'].applicationType == 'uaa'`
we put this on a new function above: _getAppFolders()